templates/web/pages/rsvp.tmpl

```diff
@@ -122,11 +119,77 @@
               </div>
             </div>
 
-            <p>Please join our Slack workplace to meet other hackers and hear announcements from the team!
-            At this time, we are not able to provide buses or travel reimbursements. We encourage joining the #carpool channel on Slack to find others from your school.</p>
+            <h4>Address (for swag)</h4>
 
-            <div class="field has-text-centered">
-              <a class="button is-red is-outlined" href="https://join.slack.com/t/boilermakevii/shared_invite/enQtODg5NjE1NDQ5MTczLWJhZTA3MGRlMWI0ZjVjNjU4YjIyN2Y3NDAxOWIwNTEwMGFmNjJhZTljYTI1MmM2MWNkYmE0ZmJmYTRlMDUyNzk" target="_blank">Join BM VII Slack</a>
+            <div class="field is-horizontal">
+              <div class="field-label is-normal">
+                <label for="accommodations" class="label">Street Address</label>
+              </div>
+              <div class="field-body">
+                <div class="field">
+                  <div class="control">
+                    <input type="text" name="street-address" class="input" maxlength="1024" value="{{ .FormRefill.StreetAddr}}">
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="field is-horizontal">
+              <div class="field-label is-normal">
+                <label for="accommodations" class="label">City</label>
+              </div>
+              <div class="field-body">
+                <div class="field">
+                  <div class="control">
+                    <input type="text" name="city" class="input" maxlength="1024" value="{{ .FormRefill.City}}">
+                  </div>
+                </div>
+              </div>
+            </div>
+
+             <div class="field is-horizontal">
+               <div class="field-label is-normal">
+                 <label for="accommodations" class="label">State</label>
+               </div>
+               <div class="field-body">
+                 <div class="field">
+                   <div class="control">
+                     <input type="text" name="state" class="input" maxlength="1024" value="{{ .FormRefill.State}}">
+                   </div>
+                 </div>
+               </div>
+             </div>
+
+            <div class="field is-horizontal">
+              <div class="field-label is-normal">
+                <label for="accommodations" class="label">Country</label>
```

Comment:
Hmm maybe this could be confusing because we only ship swag within the US (at least I think that's the only place we ship to). If that's the case then maybe we just get rid of this field?

Reply:
Let me double check with UX

Reply:
They want us to keep it, but specify we can only ship within the US